app/grandchallenge/evaluation/models.py

```diff
@@ -475,6 +483,25 @@ class Phase(FieldChangeMixin, HangingProtocolMixin, UUIDModel):
         blank=True,
         help_text="The output interfaces that the algorithms for this phase must use",
     )
+    selectable_gpu_type_choices_evaluation: list[str] = models.JSONField(
+        blank=True,
+        null=True,
+        default=list,
```

Comment:
Maybe the default choices should be `lambda: [GPUTypeChoices.NO_GPU, GPUTypeChoices.T4]` as discussed in the other comment?

Reply:
Ah, good to know!